rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1278,6 +1278,29 @@ JobIdData submit(@HeaderParam("sessionid") String sessionId, @PathParam("path")
             MultipartFormDataInput multipart, @Context UriInfo contextInfos) throws JobCreationRestException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)